### PR TITLE
add_comments_http_response

### PR DESCRIPTION
**Pull request by nitinmohan87**
Fixes aliaser tool when generating aliases for HTTP DSL

### CODE REVIEW

**Comment by raphael** [comment]
Thanks for the PR! The idea of a linter is to help find inconsistencies, it shouldn't force nonsensical code or comments :-) The comments were done that way intentionally. Would you mind putting them back but keeping the other fixes this PR contains? Thank you!

File: cmd/aliaser/main.go [hunk 1]
```diff
@@ -295,7 +295,9 @@ import (
 	// alias implementations.
 	aliasConstT = `const (
 {{- range $i, $n := .Names}}
+{{- $commLen := len $.Comments }}{{- if gt $commLen 0 }}
```
Comment (raphael) [inline cmd/aliaser/main.go]: I believe this can be simply expressed as:
```go
{{ if $.Comments }}
```

File: design/mapped_attribute.go [hunk 2]
```diff
@@ -152,3 +152,16 @@ func (ma *MappedAttributeExpr) Merge(other *MappedAttributeExpr) {
 		}
 	}
 }
+
+// FindKey finds the given key in the mapped attribute expression.
```
Comment (raphael) [inline design/mapped_attribute.go]: Is this supposed to be in this PR?

Comment (nitinmohan87) [reply design/mapped_attribute.go]: I can create a separate PR if you would like. Just thought I would add this here so that we can fix the plugins build.

**Comment by raphael** [comment]
Thank you!

**Pull request merged**